app-android/tutashared/src/main/java/de/tutao/tutashared/ModuleResources.kt

```diff
@@ -0,0 +1,21 @@
+package de.tutao.tutashared
+
+import android.util.Log
+
+class ModuleResources {
```

Comment:
similarly with this, can't shared module have its own `src/values`?

Reply:
We can, the only thing is that the resource will be duplicated three times, if it isn't a problem, I can manage to generate the needed translations into tutashared